tests/test_core.py

```diff
@@ -7845,6 +7846,7 @@ def setUp(self):
 
     assert emcc_args is not None
     self.emcc_args = emcc_args[:]
+    Settings.load(self.emcc_args)
```

Comment:
why is this needed now? is it part of the fix, or unrelated?

Reply:
I see, thanks. lgtm!

Reply:
Are you sure this is correct?  The old code seems to populate the settings_mods, which gets passed to emcc subproccess on the command line.  This new code seems to modify the test runners in-memory copy of the Settings only.

Reply:
Oh I see, it doesn't matter since those emcc args will be passed to emcc subprocess on the command line anyway.  

LGTM.